pkg/appmanager/resourceConfig.go

```diff
@@ -1450,12 +1450,18 @@ func (appMgr *Manager) createRSConfigFromIngress(
 	defaultIP,
 	snatPoolName string,
 ) *ResourceConfig {
+	// if manage-default-ingress is false, then process ingress with ingress.class=f5 annotation only
```

Comment:
Please change in comment "manage-default-ingress" to "manage-ingress-class-only"

Reply:
sure.